app/misc/authenticate.ts

```diff
@@ -20,18 +20,69 @@ var repoNotFound = 0;
 var signed = 0;
 var changes = 0;
 let signedAfter = false;
-let loginScopes = [
-  "repo",
-  "user"
-];
 
+let oauthpass = 'x-oauth-basic';
+let electronOauth2 = require('electron-oauth2');
+let account;
+
+//config from oauth library
+const OauthConfig = {
+  clientId: 'c2509d8769f5f1e46028',
+  clientSecret: 'e4717832659e95c2f8f4237a3390cf09013b94f7',
+  authorizationUrl: 'https://github.com/login/oauth/authorize',
+  tokenUrl: 'https://github.com/login/oauth/access_token',
+  useBasicAuthorizationHeader: false,
+  redirectUri: 'http://localhost:8080/user/signin/callback'
+};
+
+const windowParams = {
+  alwaysOnTop: true,
+  autoHideMenuBar: true,
+  webPreferences: {
+    nodeIntegration: false,
```

Comment:
Node Integration false? Pretty sure we need this to be true? Can you comment why you made this false?

Reply:
It is recommended for security reasons to not enable nodeIntegration in the Oauth request window.